src/app/shared/services/auth.service.ts

```diff
@@ -0,0 +1,44 @@
+import { userObject } from './../interfaces/userObject';
+import { MessageService } from '@shared/services/message.service';
+import { Injectable } from '@angular/core';
+import { LoginObject } from '@shared/interfaces/loginObj';
+import { Observable, BehaviorSubject, of } from 'rxjs';
+import { HttpClient, HttpHeaders } from '@angular/common/http';
+// import { FormingApiUrlService } from './auxiliaryServices/forming-api-url.service';
+// import { catchError } from 'rxjs/operators';
+
+const httpOptions = {
+  headers: new HttpHeaders({ 'Content-Type': 'application/json' })
+};
+
+@Injectable({
+  providedIn: 'root'
+})
+
+export class AuthService {
+  constructor(private http: HttpClient, private messageService: MessageService) {}
+  token = localStorage.getItem('token');
+  checkLogIn = new BehaviorSubject<boolean>(this.token ? true : false);
+  currentLoginStatus = this.checkLogIn.asObservable();
```

Comment:
why you use asObservable(); explicitly?

Reply:
also, you can use directly checkLogIn, no need an additional variable here

Reply:
I have questions here

Reply:
I've made it with getter and setter